sass/themes/frost/components/_progress_indicator.scss

```diff
@@ -0,0 +1,125 @@
+// ==========================================================================
+// Progress indicator
+// ==========================================================================
+
+$circle-size: 15px !default;
+$circle-line-thickness: 3px !default;
+$indicator-thickness: 2px !default;
+
+.flexer {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -webkit-flex;
+    display: flex;
+}
+.flexer-element {
+    -ms-flex: 1;
+    -webkit-flex: 1;
+    -moz-flex: 1;
+    flex: 1;
+}
+
+.progress-indicator {
+    @extend .flexer;
+    margin: 0;
+    padding: 0;
+    text-transform: capitalize;
+    margin-bottom: 20px;
+    padding-bottom: 20px;
+    border-bottom: 1px solid $colour-border-grey;
+
+    > li {
+        @extend .flexer-element;
```

Comment:
@KoalaMango it feels a bit over qualifying the selectors here. it will be difficult to override them if we need to. I'd recommend moving `.progress-indicator__circle` to the root (outside `.progress-indicator`).